Formula/qt-ios.rb

```diff
@@ -0,0 +1,76 @@
+class QtIos < Formula
+  desc "Qt for iOS"
+  homepage "https://www.qt.io/"
+  url "https://download.qt.io/official_releases/qt/6.0/6.0.3/single/qt-everywhere-src-6.0.3.tar.xz"
+  sha256 "ca4a97439443dd0b476a47b284ba772c3b1b041a9eef733e26a789490993a0e3"
+  license all_of: ["GFDL-1.3-only", "GPL-2.0-only", "GPL-3.0-only", "LGPL-2.1-only", "LGPL-3.0-only"]
+
+  keg_only "this is the SDK of Qt for iOS"
```

Comment:
It's weird that this `depends_on qt` but also installs the same binaries (and/or libraries).

Can we rename things so that both `qt` and `qt-ios` can be linked into the prefix?

Reply:
There is an option `-qtlibinfix <infix> .. Rename all libQt6*.so to libQt6*<infix>.so.`, but not sure for the `include` dirs.

From [blog post](https://www.qt.io/blog/qt-6-build-system), it need run native host tools. Programs/scripts in `bin` also have the same name with `qt`, but `Qt Creator` can distinguish them.

Maybe I can move the programs to `libexec` (this is what the future version of `qt` do) and create a launch script.